e2e/src/suite/cancelResume.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as assert from "assert"
+
+import { sleep, waitForToolUse, waitForMessage } from "./utils"
+
+suite("Roo Code Cancel & Resume", () => {
+	test.skip("Should handle subtask cancellation and resumption correctly", async function () {
```

Comment:
Skipping this for now until we can fix it... CC @mrubens @cannuri

Reply:
Sounds good, it's been flaking for me